aleph/index/entities.py

```diff
@@ -201,7 +202,16 @@ def delete_entity(entity_id, exclude=None, sync=False):
         index = entity.get('_index')
         if index == exclude:
             continue
-        es.delete(index=index, id=entity_id,
-                  refresh=refresh_sync(sync))
-        q = {'term': {'entities': entity_id}}
-        query_delete(entities_read_index(), q, sync=sync)
+        try:
+            es.delete(index=index, id=entity_id,
+                      refresh=refresh_sync(sync))
+            q = {'term': {'entities': entity_id}}
+            query_delete(entities_read_index(), q, sync=sync)
+        except NotFoundError:
```

Comment:
For me to understand, the goal here is explicitness?

Reply:
Yes, for explicitness and we also want to skip the `query_delete` step which is slow and doesn't do anything where the entity is already deleted once